player.py

```diff
@@ -1,28 +1,56 @@
 import random
+import csv
+
 #from . import checkpointManager
 
 class Player:
     miles = random.randint(16,32)
     foodDaily = random.randint(12,15)
 
-    def __init__(self, character:str, money:int, food:int, huntAdjust:float, buyHudjust:float, distNext:int):
+    def __init__(self, character:str, money:int, food:int, huntAdjust:float, buyAudjust:float, distNext:int):
         self.character = character
         self.money = money
         self.food = food
         self.huntAdjust = huntAdjust
-        self.buyAdjust = buyHudjust
+        self.buyAdjust = buyAudjust
```

Comment:
@christopherhochgesang You should use this variable in the river scenario instead of using a static adjustment

Reply:
It's not a static adjustment. It is character based depending on the selected character just as the huntAdjust is, and therefore an attribute of the player not the scenario.

Reply:
Yes but Chris has a line that didn't use this. So he should. Sorry this comment was for him
```python
# Pay for ferry choice Choice
                    elif(playerChoice==2):
                        if(self.player.character=="Merchant"):
                            ferryCost = self.ferryCost*.9
```